hurumap/settings.py

```diff
@@ -274,6 +274,13 @@
      'https://www.youtube.com/embed/7mSZnXFHFxo?list=PL7MJ_sFHs952CYcKHPQp786HVVy83nBwH')
 ])
 
+# color scheme
+HURUMAP['theme'] = {
```

Comment:
Lets call this `charts` to differentiate it from general theme/look and feel of HURUmap @phillipahereza (or at least `HURUmap['theme']['charts']`)